modmesh/gui/naca.py

```diff
@@ -108,37 +107,15 @@ def draw_naca4(world, number, npoint, fac, off_x, off_y, **kw):
     return crds
 
 
-def app_main():
+def runmain():
     """
     A simple example for drawing a couple of cubic Bezier curves.
     """
     w = core.WorldFp64()
     draw_naca4(w, number='0012', npoint=101, fac=5.0, off_x=0.0, off_y=2.0,
                open_trailing_edge=False, cosine_spacing=True)
-    return w
-
-
-def load_app():
```

Comment:
Turn the example NACA 4-digit airfoil shape a simple function.  An "app" is an overkill.